adminimize.php

```diff
@@ -1708,3 +1714,46 @@ function _mw_adminimize_install() {
 	}
 	wp_cache_add( 'mw_adminimize', $adminimizeoptions );
 }
+
+/**
+ * Make sure adminimize option is complete when a role json file is imported
+ */
+function _mw_adminimize_roles_complete_options( $roles_options ){
+
+	$adminimizeoption = _mw_adminimize_get_option_value();
+
+	foreach ( $roles_options as $role_option_name => $role_option_value ){
+		$adminimizeoption[$role_option_name] = $role_option_value;
+	}
+
+	return $adminimizeoption;
+}
+
+/**
+ * Check if options comes from roles adminimize settings export
+ */
+function _mw_adminimize_is_roles_options_import( $options ){
+	global $wp_roles;
+
+	$roles_options = [];
+	foreach (  $wp_roles->role_names as $role_slug => $role_name ){
+
+		$role_options = array_filter(
+			$options, function ( $option_key ) use ( $role_slug ) {
+				return stripos( $option_key, '_' . $role_slug ) !== false;
+			}, ARRAY_FILTER_USE_KEY
+		);
+
+		if ( empty( $roles_options ) ){
+			$roles_options = $role_options;
+		} else {
+			$roles_options = array_merge( $roles_options, $role_options );
+		} 
+	}
+
+	if ( count( $options ) == count( $roles_options ) ){
+		return true;
+	} else {
```

Comment:
This else should not necessary. `if`is enough than return true or if the `if` fails is the `return false`good enough.